mm2src/coins/eth.rs

```diff
@@ -6358,6 +6450,16 @@ async fn get_max_eth_tx_type_conf(ctx: &MmArc, conf: &Json, coin_type: &EthCoinT
                 }
             }
         },
+        EthCoinType::Trx => {
+            // Tron doesn't use Ethereum's typed transactions (EIP-2718)
+            // Tron has its own transaction format and fee mechanism,
+            // not EIP-1559 style transactions with baseFeePerGas
+            warn!(
```

Comment:
prolly wanna mark this with a todo

Reply:
done https://github.com/KomodoPlatform/komodo-defi-framework/pull/2467/commits/1f69c9d0e62b5e8f996f29f04c50a51a2e6baec8